AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -48,7 +48,7 @@ public final class GiftCardComponent: PresentableComponent,
     public let style: FormComponentStyle
 
     /// A boolean value that determines whether the payment button is displayed. Defaults to `true`.
-    public private(set) var showSubmitButton: Bool
+    internal let showSubmitButton: Bool
```

Comment:
Very nitpicky but what about 
```suggestion
    internal let showsSubmitButton: Bool
```

Reply:
Replaced everywhere. I saw where using the third form in other property names too. Thanks